src/protocols/strategies/convex.strategy.ts

```diff
@@ -287,5 +287,10 @@ async function retrieveBribesProcessorData(chain: Chain, vault: VaultDefinition)
   const timestampCutoff = Math.floor(Date.now() / 1000 - 21 * ONE_DAY_SECONDS);
   const { data } = await evaluateEvents(harvests, distributions, { timestamp_gte: timestampCutoff });
 
-  return estimateVaultPerformance(chain, vault, data);
+  try {
+    return estimateVaultPerformance(chain, vault, data);
```

Comment:
this is due to atm the processor only has 1 event and can't establish a length of time - it will be fixed on the next processor event as only 2 are required